### DISCUSSION TITLE
How to read and plot a NIFTI / .nii file and plot it

### DISCUSSION THREAD

**Discussion by lucasgoddamn** [pyvista/pyvista#2971] [Q&A] [answered]
Hey, im working on MRI Data and im quite new to programming and discovered pyvista for my purposes. 
Since you cant read NIFTI files with pv.read() i was wondering how i could transform my NIFTI files to a VTK or PLY and then plot it
Even though this might seem simple i haven't found any working solution yet, so i appreciate any quick help!

**Comment by MatthewFlamm** [comment, marked as answer]
Pyvista does not yet support NIFTI files, but it looks vtk does:  https://vtk.org/doc/nightly/html/classvtkNIFTIImageReader.html

This worked for me, and I'm going to make a feature request issue for this.

```py
import pyvista as pv
from vtk import vtkNIFTIImageReader
class NIFTIReader(pv.BaseReader):
    _class_reader = vtkNIFTIImageReader
reader = NIFTIReader('path_to_file.nii.gz')
mesh = reader.read()
mesh.plot()
```

**Reply by MatthewFlamm** [reply]
#2972 has the feature request.
